src/components/inputs/TokenInput/TokenInput.vue

```diff
@@ -250,9 +283,21 @@ watchEffect(() => {
         <BalProgressBar
           v-if="hasBalance && !noMax"
           :width="maxPercentage"
+          :buffer-width="bufferPercentage"
           :color="barColor"
           class="mt-2"
         />
+        <div
+          v-if="shouldShowTxBufferMessage"
+          class="mt-2 text-yellow-600 dark:text-yellow-400 text-sm"
```

Comment:
Text should be `xs` to match BalTextInput error text size.
```suggestion
          class="mt-2 text-yellow-600 dark:text-yellow-400 text-xs"
```